doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -81,62 +81,63 @@ The Kubernetes Node OS (as with any OS) consists of two main components:
-etc.), and the kernel features required to provide the isolation mechanisms
-(cgroups, namespaces, filesystems, etc.) between the components.
+running in the user space (which usually has its own southbound interface in an
+interpreter or libraries). Key containerization capabilities, such as control
+groups (cgroups) and namespaces, are kernel features, and are used and managed
+by the container runtime, in order to provide isolation between the user space
+processes. This would also include the container itself, as well as any processes
+running within it. The security of the Kubernetes Node OS and its relationship
+to the container and the applications running within the container, or containers,
+is essential to the overall security posture of the entire system, and must be
+appropriately secured to ensure that the processes running in one container cannot
+escalate their privileges or otherwise affect processes running in an adjacent
+container. An example of this concept, together with further details, can be found
+in :ref:`chapters/chapter06:api and feature testing requirements`.
+
+It is important to note that the container runtime itself is also a set of processes
+that run in user space, and therefore also interact with the kernel via system calls.
+Many diagrams show containers as running on top of the runtime, or inside the runtime.
+More accurately, the containers themselves are simply processes running within an OS.
+The container runtime is simply another set of processes that are used to manage
+these containers (pull, run, delete, and so on) and the kernel features required
+to provide the isolation mechanisms (cgroups, namespaces, filesystems, and so on)
+between the components.
 
 Container Runtime Services
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Container Runtime is the component that runs within a Kubernetes Node
-Operating System (OS) and manages the underlying OS functionality, such as
+The Container Runtime is a component that runs within a Kubernetes Node
+Operating System (OS). It manages the underlying OS functionality, such as
 cgroups and namespaces (in Linux), in order to provide a service within which
-container images can be executed and make use of the infrastructure resources
-(compute, storage, networking and other I/O devices) abstracted by the Container
-Host OS, based on API instructions from the kubelet.
-
-There are a number of different container runtimes. The simplest form, low-level
-container runtimes, just manage the OS capabilities such as cgroups and
-namespaces, and then run commands from within those cgroups and namespaces. An
-example of this type of runtime is runc, which underpins many of the
-higher-level runtimes and is considered a reference implementation of the `Open
+container images can be executed. It also makes use of the infrastructure
+resources, such as compute, storage, networking and other I/O devices,
+abstracted by the Container Host OS, based on API instructions from the kubelet.
```

Comment:
Container Host OS or Kubernetes Node Host OS?

Reply:
> Container Host OS or Kubernetes Node Host OS?

Fixed in [8eeb6e9](https://github.com/cntt-n/CNTT/pull/3217/commits/8eeb6e9c166cc3a4166238034dc17583f2e5fbac)